library/src/main/res/layout/exo_simple_player_view.xml

```diff
@@ -36,6 +36,12 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent"/>
 
+    <ImageView
```

Comment:
This doesn't generalize properly to support multiple image cues at once. Any solution will likely need SubtitleView/SubtitlePainter to handle drawing directly with proper support for this, similarly to what happens in the text case.